apps/openmw/mwworld/containerstore.cpp

```diff
@@ -420,6 +420,17 @@ int MWWorld::ContainerStore::remove(const std::string& itemId, int count, const
     return count - toRemove;
 }
 
+bool MWWorld::ContainerStore::isEmpty() const
```

Comment:
If someone is going to use this function in future (instead of doing something like comparing the begin iterator to the end iterator), there might be issues caused by considering not actually empty containers without items visible in the inventory in them non-empty. Maybe it should be renamed to reflect its purpose better.